modules/dynamic-plugins/rhdh-supported-plugins.adoc

```diff
@@ -30,7 +31,7 @@ a|
 
 |Disabled
 
-|AAP |Backend |@janus-idp/backstage-plugin-aap-backend | |1.5.5 |Red Hat Tech Preview
+|aap |https://www.npmjs.com/package/@janus-idp/backstage-plugin-aap-backend[@janus-idp/backstage-plugin-aap-backend] |backend-plugin |1.6.13 |Red Hat Tech Preview
```

Comment:
Is it possible to generate the plugin name using the same format as we have used previously? I think this is more readable and would maintain consistency with what we have published previously.

Also, for consistency, could we parse `frontend-plugin` and `backend-plugin` to read 'Frontend` / `Backend`?

Reply:
I'd suggest leaving the plugin names as you have generated them in the script, I don't think we can come up with a comprehensive rule for reformatting the names e.g. aap => AAP, azure-devops => Azure Devops, sonarqube => SonarQube.

I think differentiating plugins from plugin-modules may be adding unnecessary detail, so I'd leave it as `Frontend` or `Backend` as we have it in 1.1, for consistency.

Reply:
OK, I'll make the change to just call out front or back, rather than the 4 types of plugin.

Reply:
> I don't think we can come up with a comprehensive rule for reformatting the names

turns out with a few exceptions, it's easy... https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/302/files#diff-37a9a4a97efdedba99d8a292642e60bbf89e059d57617e1a98ee786dd4814817R60-R78